src/components/ListCounter.jsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+const ListCounter = props => {
```

Comment:
This is not blocking or at all perssing. 

In general these display only type components can be written with the props destructured and then you can return the jsx imeediately ([implicit return](https://jaketrent.com/post/javascript-arrow-function-return-rules/))



```jsx
const ListCounter = ({ count, total}) => (
  <div className="dg_service_counter">
    <p>{`Showing ${count} of ${total} services`}</p>
  </div>
);
```

and i suspect maybe even further but not sure to

```jsx
const ListCounter = ({ count, total}) => (
  <p className="dg_service_counter">{`Showing ${count} of ${total} services`}</p>
);
```

Reply:
done